source/ed/osm2ed.cpp

```diff
@@ -907,11 +922,42 @@ void OSMCache::flag_nodes() {
 
 }}
 
+std::map<std::string, std::string> parse_poi_types(const std::vector<std::string>& poi_types){
+    std::map<std::string, std::string> result;
+    for(auto type: poi_types){
+        std::vector<std::string> strs;
+        boost::algorithm::split(strs, type, boost::is_any_of("="));
+        if(strs.size() == 1){
+            result[strs[0]] = strs[0];
+        }else if(strs.size() == 2){
+            result[strs[0]] = strs[1];
+        }else{
+            std::cout << "poi_type: \"" << type << "\" ignored!" << std::endl;
```

Comment:
it's used for parse the command line input, others case like this (--help, --version) output on cout too.
It seemed more logic to used cout for this too.